shoop/admin/modules/products/views/forms.py

```diff
@@ -263,3 +284,23 @@ def save(self, commit=True):
 class ProductImageMediaFormSet(ProductMediaFormSet):
     allowed_media_kinds = [ProductMediaKind.IMAGE]
     form_class = ProductImageMediaForm
+
+    def save(self, commit=True):
+        """
+        Add first saved image as  primary image for this product by default if none is selected as such
```

Comment:
@akx has always said that there is no point documenting the obvious. I thought that mentioning that `save` actually saves the form goes into that category :)

Agree though that the title is missing and was thinking that when I made the docstring. Will fix :)